monkey/monkey_island/cc/next_ui/src/app/(protected)/plugins/installed/page.tsx

```diff
@@ -35,7 +42,7 @@ export default function InstalledPluginsPage() {
             (installedPlugin) => installedPlugin.id === row.id
         );
         if (!plugin) return [];
-        return [...getUpgradeAction(plugin), ...getUninstallAction(plugin)];
+        return [getUpgradeAction(plugin)];
```

Comment:
Uninstall got removed?

Reply:
It's empty anyways, it will get added later